packages/nextjs/app/builders/0x8eE31084d2914fA84Baae3460093564934837898/page.tsx

```diff
@@ -0,0 +1,94 @@
+import Image from "next/image";
+import { NextPage } from "next";
+import type { Metadata } from "next";
+import {
+  AcademicCapIcon,
+  CodeBracketIcon,
+  RocketLaunchIcon,
+  StarIcon,
+  TrophyIcon,
+  XMarkIcon,
+} from "@heroicons/react/24/outline";
+import { BuidlGuidlLogo } from "~~/components/assets/BuidlGuidlLogo";
+import { Address } from "~~/components/scaffold-eth";
+import { getMetadata } from "~~/utils/scaffold-eth/getMetadata";
+
+const page: NextPage = () => {
+  return (
+    <>
+      <div className="w-full max-w-6xl mx-auto rounded-3xl shadow-2xl flex flex-row my-10 animate-gradient-x">
+        {/* image column */}
+        <div className="w-1/2 rounded-l-3xl bg-secondary">
+          <div className="flex justify-center items-center h-full">
+            <div className="w-80 h-80 overflow-hidden rounded-full shadow-lg">
+              <Image
+                src="/builders/pena.jpeg"
```

Comment:
No image is showing, the path is wrong, it should be `/builders/avatars/pena.jpeg`